universal-login-sdk/src/core/services/WalletService.ts

```diff
@@ -128,7 +128,8 @@ export class WalletService {
   async recover(name: string, passphrase: string) {
     const contractAddress = await this.sdk.getWalletContractAddress(name);
     const wallet = await this.walletFromPassphrase(name, passphrase);
-    ensure(await this.sdk.keyExist(contractAddress, wallet.address), InvalidPassphrase);
+    const deployedWallet = new DeployedWallet(contractAddress, name, '', this.sdk);
```

Comment:
this.getDeployedWallet();

Reply:
'UNIT: WalletService' test doesn’t work with this.getDeployedWallet();